engine/runtime/src/main/scala/org/enso/interpreter/runtime/IrToTruffle.scala

```diff
@@ -2228,4 +2275,13 @@ class IrToTruffle(
           )
       }
   }
+
+  private def asScope(module: CompilerContext.Module): ModuleScope = {
```

Comment:
nit: `as` suggests that we have some casting involved, `get` or `infer` would seem to suggest that we get the scope for the given module.

Reply:
>  some casting involved

In such case I have chosen the right name. There is casting hidden in `fromCompilerModule` and it could fail if some non-`TruffleCompilerContext.Module` is passed in.